vital/algo/read_track_descriptor_set.h

```diff
@@ -43,12 +46,14 @@ class VITAL_ALGO_EXPORT read_track_descriptor_set
   static std::string static_type_name() { return "read_track_descriptor_set"; }
 
   /// Open a file of track descriptor sets.
+
   /**
    * This method opens a track descriptor set file for reading.
    *
    * \param filename Name of file to open
    *
-   * \throws kwiver::vital::path_not_exists Thrown when the given path does not exist.
+   * \throws kwiver::vital::path_not_exists Thrown when the given path does not
+   * exist.
```

Comment:
Okay, I guess we decided we're punting on Doxygen formatting for now?

Reply:
This is reformatted now.  See comment on detected_object_set_input.h above.

Reply:
FYI, there is still a mix of better and not-so-great Doxygen formatting, though again, I'm not sure we necessarily want to deal with it here.